bundles/org.dataflowanalysis.converter/src/org/dataflowanalysis/converter/PCMConverter.java

```diff
@@ -324,7 +389,28 @@ private void addNodeCharacteristicsToNode(Node node, List<CharacteristicValue> c
             }
         }
     }
+    
+    /**
+     * Get or create a LabelType with the supplied String name
+     * @param name the name of the LabelType
+     * @return the LabelType
+     */
+    private LabelType getOrCreateLabelType(String name) {
+    	 LabelType type = dataDictionary.getLabelTypes()
+                 .stream()
+                 .filter(f -> f.getEntityName()
+                         .equals(name))
+                 .findFirst()
+                 .orElseGet(() -> createLabelType(name));
+    	 
+    	 return type;
+    }
 
+    /**
+     * Get or create a Label from the supplied CharacteristicValue
+     * @param charValue the CharacteristicValue
+     * @return the Label
+     */
     private Label getOrCreateDFDLabel(CharacteristicValue charValue) {
```

Comment:
Is this function still used? Because you implemented a second version of it below, that achieves the same

Reply:
One works on `CharacteristicValues` (e.g. from TFGs), while the other uses a string for the label and `LabelType` for the specified type (e.g. values in DFDs)

Reply:
Fixed in 9161111817fd